src/ExtInterface/ReefMod/ReefModDomain.jl

```diff
@@ -194,15 +194,24 @@ end
 function load_initial_cover(
     ::Type{ReefModDomain},
     dom_data::Dataset,
-    location_data::DataFrame,
+    site_data::DataFrame,
     loc_ids::Vector{String},
     init_yr::Int=2022
 )::YAXArray
     if !haskey(dom_data.cubes, :coral_cover_per_taxa)
         @error "coral_cover_per_taxa variable not found in ReefMod data"
     end
-    init_cc_per_taxa::YAXArray = Cube(dom_data[["coral_cover_per_taxa"]])[time=At(init_yr)]
 
+    c_axes = caxes(dom_data.coral_cover_per_taxa)
```

Comment:
You can leave it this way if you want but you could also do something like:

```
cover_axes_names = ADRIA.axes_names(dom_data.coral_cover_per_taxa) 
cover_axes = ADRIA.axis_labels.([dom_data.coral_cover_per_taxa], cover_axes_names)
```

And we could also write a `ADRIA.axes_labels` that accepts just the datacube and returns an array of labels for all axes. This way you don't have to do this `Int64.(collect(..))`

Reply:
I believe this is done because of an issue with DiskArrays and Floating point axis labels.